.github/workflows/helm-chart-ci.yaml

```diff
@@ -209,6 +197,7 @@ jobs:
             "${TEST_DIR}/install.sh"
           else
             ct install --debug \
+              --charts "charts/spire" \
```

Comment:
Can I get a simple explanation of why this line needed to be added to "always run all the tests"?

Seems like if it is necessary, there was / is an undocumented bug in our testing.

Reply:
From a chat conversation with Kevin, without the `--charts` parameter, the software itself determines that the charts are not requiring an update, so the tests would be effectively skipped without this extra parameter

Closing this question as resolved.